css-box-model/styles.css

```diff
@@ -0,0 +1,11 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+  margin-right: 12px;
+  margin-left: 12px;
+  padding-top: 6px;
+  padding-bottom: 6px;
+  border-radius: 6px;
+
+  /* your code here 👇 */
+}
```

Comment:
Where's the side padding?

---

![image](https://user-images.githubusercontent.com/16143771/167919652-0810546f-78cf-4c63-86e4-a3e5b087ac7d.png)

---

Also:

---

![image](https://user-images.githubusercontent.com/16143771/167919734-07308872-f3b2-4bed-91d1-00907b74e3ca.png)